src/js/utils/form-values-utils.jsx

```diff
@@ -52,3 +52,12 @@ export const getReceivingPayloadContainers = formValues =>
       return _.omit(item, 'product.displayNames');
     }),
   }));
+
+/** Expects to have filterValue already in lower case */
```

Comment:
Would not it be clearer if everything was lowered in one place?

Reply:
When I look at it now, you are right - I just wanted to avoid writing three times `filterValue.toLowerCase()` here, but I will just create a new const on the top of the function which will make `filterValue.toLowerCase()`.
Thanks for the advice